lib/offsite_payments/integrations/molpay.rb

```diff
@@ -60,8 +60,7 @@ def form_fields
         end
 
         def amount=(money)
-          #Molpay minimum amount is 1.01
-          if money.is_a?(String) or money.to_f < 1.01
+          if money.is_a?(String)
```

Comment:
why not test for what we want ? as in `unless money.is_a?(Money) || money <= 0 `